_overviews/core/custom-collection-operations.md

```diff
@@ -81,11 +107,25 @@ The type `IsIterable[Repr]` has implicit instances for all types `Repr` that can
 to `IterableOps[A, Iterable, C]` (for some element type `A` and some collection type `C`). There are
 instances for actual collection types and also for `String` and `Array`.
 
+{% endtab %}
+{% tab 'Scala 3' for=sumBy_3 %}
+
+We expect the `sumBy` to work on any type that is *like* a collection, such as `String`
+and `Array`. Fortunately, the type `IsIterable[Repr]` has implicit instances for all types `Repr` that can be converted
+to `IterableOps[A, Iterable, C]` (for some element type `A` and some collection type `C`) and there are
+instances for actual collection types and also for `String` and `Array`.
+
```

Comment:
I think it will be useful to show how to change the code example to support this:
```scala
import scala.collection.generic.IsIterable

extension [Repr](repr: Repr)(using iter: IsIterable[Repr])
  def sumBy[B: Numeric](f: iter.A => B): B =
    val coll = iter(repr)
    val it = coll.iterator
    var result = f(it.next())
    while it.hasNext do
      result = summon[Numeric[B]].plus(result, f(it.next()))
    result
```

Reply:
You're right.